tests/analysis/Decompilation/dynamic_native/array.c

```diff
@@ -0,0 +1,35 @@
+#include <stdint.h>
+
+typedef struct __attribute__((packed)) {
+  uint16_t uint16_array[10];
+  uint64_t uint64;
+} StructT;
```

Comment:
What's the trailing `T` for? Maybe `TestStruct`?

Reply:
fixed